src/domain/resource/card/ResourceCard.js

```diff
@@ -187,9 +189,14 @@ class ResourceCard extends React.Component {
             }
             text={
               accessibilityShortcomingsCount
-                ? t('ResourceInfo.foundAccessibilityShortcomings', {
-                  nShortcomings: accessibilityShortcomingsCount,
-                })
+                ? t(
+                  accessibilityShortcomingsCount === 1
+                    ? 'ResourceInfo.foundAccessibilityShortcoming'
+                    : 'ResourceInfo.foundAccessibilityShortcomings',
+                  {
+                    nShortcomings: accessibilityShortcomingsCount,
+                  },
+                )
```

Comment:
Wondering whether `react-intl` has an official way to handle pluralization.

Reply:
There are components, but I don't think there's a simpler way with the `t` function. In any case you need to provide the translations manually for singular and plural forms.